### PR TITLE
Update windows-autopilot-hybrid.md

### PR DESCRIPTION
**Pull request by msbemba**
Tested and removing GA role after the installation still works

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/1444

**Comment by PRMerger10** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: memdocs/autopilot/windows-autopilot-hybrid.md [hunk 1]
```diff
@@ -126,10 +126,14 @@ The Intune Connector requires the [same endpoints as Intune](../intune/fundament
 4. Open the downloaded Connector setup file, *ODJConnectorBootstrapper.exe*, to install the Connector.
 5. At the end of the setup, select **Configure**.
 6. Select **Sign In**.
-7. Enter the user Global Administrator or Intune Administrator role credentials. 
+7. Enter the Global Administrator or Intune Administrator role credentials.
```
Comment (JohanFreelancer9) [inline memdocs/autopilot/windows-autopilot-hybrid.md:129]:
```suggestion
7. Enter the Global administrator or Intune administrator role credentials. 
```

File: memdocs/autopilot/windows-autopilot-hybrid.md [hunk 2]
```diff
@@ -126,10 +126,14 @@ The Intune Connector requires the [same endpoints as Intune](../intune/fundament
...
 6. Select **Sign In**.
-7. Enter the user Global Administrator or Intune Administrator role credentials. 
+7. Enter the Global Administrator or Intune Administrator role credentials. 
  The user account must have an assigned Intune license.
 8. Go to **Devices** > **Windows** > **Windows enrollment** > **Intune Connector for Active Directory**, and then confirm that the connection status is **Active**.
 
+> [!NOTE]
+> The Global administrator role is a temporary requirement at the time of installation.
+
+
```
Comment (JohanFreelancer9) [inline memdocs/autopilot/windows-autopilot-hybrid.md:136]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by PRMerger16** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger18** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by ErikjeMS** [comment]
@greg-lindsay Can you review this autopilot PR? Thanks!

**Comment by yogkumgit** [comment]
@dougeby Kindly help.

**Pull request merged**